generators/path_generator

```diff
@@ -32,20 +33,23 @@ matches = []
 for cfg in glob.glob(os.path.join(conf_dir, 'generators', 'meta-path',
                                   '*.json')):
     with open(cfg, 'r') as jf:
+        global_extra_tags = []
+        extra_code = ""
         data = json.load(jf)
         name = data['name']
         project = data['project']
         if 'should_fail_because' in data:
-            should_fail_because = ':should_fail_because: ' + data[
-                'should_fail_because']
-        else:
-            should_fail_because = ''
```

Comment:
Looks like you can remove this variable up above.

Reply:
Right! removed